airsonic-main/pom.xml

```diff
@@ -197,6 +201,15 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+		<dependency>
+			<groupId>org.checkerframework</groupId>
+			<artifactId>checker-qual</artifactId>
+			<!-- Guava uses it implicitly.
+				When PRing, it is necessary to confirm
+				whether null annotation is permitted. -->
+			<scope>provided</scope>
```

Comment:
There is no problem in ``compile``.
As a result, both are the same.

In the case of ``compile``, it is explicitly included in the product.
On the other hand, in the case of an annotation library, the option of ``provided`` is available if it is used for compilation and not included in the product.
(At present, Airsonic has no view of what to use from many nullness libraries)

The result is the same because Airsonic includes the ''provide'' library in the product for embedded.